content/en/blog/2022/prow-and-tide-for-kubernetes-contributors/prow-and-tide-for-kubernetes-contributors.md

```diff
@@ -0,0 +1,96 @@
+* How to assign the request to a specific SIG?
+* How to approve things while making it public and easily traceable?
+* How to merge a contribution without carrying all the commit messages that were created during the review?
+
+These are some of the main tasks in which Tide will help, allowing us to use the GitHub interface for these tasks (and more), making the actions more visible to the community (since they are visible as plain comments in the GitHub discussion), and allowing us to manage contributions without necessarily having to clone git repositories and manually issuing git commands.
+
+## Back to squashing
+
+One of the most common examples, and getting back to my initial one, is squashing commits: if someone makes a change in a PR, there will likely be reviews and changes, each one of them will add a new commit message. If left like this, the PR will add to the main branch all the commit messages created during the review process. To avoid this, we can _squash_ the commit messages to keep just one of them (usually, the first one): the changes will still be visible for anyone reading the PR, but they will appear as a single commit to the main branch.
+
+This can be done [through git](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History), and a cursory read of how it's done will not be obvious to someone relatively enw to git! Is there a way to avoid cloning the repository (or PR), issuing `git` commands locally, and pushing the changes?
+
+There is, with Tide. Squashing your commits is a label away and the tooling will do the rest. To do this on your PR you'll need to comment with the following:
+
+`/label tide/merge-method-squash`
+
+This will:
+
+1. Trigger Tide to squash the messages prior to merging.
+2. As a secondary effect, make your action clearly visible in the discussion section of the PR.
+
+This use of Tide is one of the most useful when submitting changes that undergo changes during the PR discussion, since it automates something to do The Right Thing (TM).
+
+There's often nothing better than an example, so let's take a look at [this proposed change to the Kubernetes website](https://github.com/kubernetes/website/pull/32685), on the topic of the `dockershim` removal FAQ. The initial commit is [followed by several others](https://github.com/kubernetes/website/pull/32685/commits), a result of the conversation and proposed reviews. The result of all those changes [is merged](https://github.com/kubernetes/website/commit/a582a21cf00c88446a7feda4effd853b108c5c9c) as a single commit, with the commit message retaining the title of the very first commit done, and the commit description being the aggregate of all the commits done in the PR. This was achieved by using `/label tide/merge-method-squat` [in a comment](https://github.com/kubernetes/website/pull/32685#issuecomment-1085801034), and did away with the need to manually rebase and/or squash using `git`: everything was possible through the GitHub interface.
```

Comment:
`/label tide/merge-method-squat` -> `/label tide/merge-method-squash`

Reply:
I do wonder if Tide could do some squats for me.

Reply:
:D